src/Commands/DeleteHost.php

```diff
@@ -21,6 +21,10 @@ public function handle()
             return $this->error("Host with name `{$name}` not found.");
         }
 
+        if (! $this->confirm("Are you sure you wish to delete '{$name}'?")) {
```

Comment:
We have the convention of surrounding a value with backticks instead of regular quotes.